pyfolio/tears.py

```diff
@@ -76,8 +77,8 @@ def create_returns_tear_sheet(returns, live_start_date=None,
         benchmark_rets = utils.get_symbol_rets('SPY')
         # If the strategy's history is longer than the benchmark's, limit
         # strategy
-        benchmark_start = benchmark_rets.index[0].astimezone(pytz.UTC)
-        returns_start = returns.index[0].astimezone(pytz.UTC)
+        benchmark_start = get_utc_date(benchmark_rets.index[0])
```

Comment:
In research all of the pricing data is UTC timestamped, I'd vote to have pyfolio do the same. The data returned by yahoo was tz-naive but this PR converts it to UTC before it's returned. I think this will fix the error we saw on staging. 

I'm also not a fan of the .astimezone() method because it's not a DatetimeIndex method. .tz_convert() and .tz_localize() work for both DatetimeIndex/Timestamp inputs.